src/local.js

```diff
@@ -14,15 +14,53 @@ module.exports = {
     return n.transpose(withBias);
   },
 
+  /**
+   * Pre-process.
+   *
+   * @param {Object} opts
+   * @param {RemoteComputationResult} opts.remoteResult
+   * @param {Project} opts.userData
+   * @returns {Object}
+   */
   preprocess(opts) {
+    const features = get(opts, 'remoteResult.pluginState.inputs[0][0]');
+
+    if (!features || !Array.isArray(features) || !features.length) {
+      throw new Error('Expected inputs containing features');
+    } else if (
+      features.some(f => FreeSurfer.validFields.indexOf(f) < 0)
+    ) {
+      throw new Error(`Unknown FreeSurfer region in inputs: ${features.toString()}`);
+    }
+
+    /**
+     * @todo: This uses only one feature. Change to use multiple!
+     *
+     * @param {FreeSurfer} freeSurfer
+     * @returns {Number}
+     */
+    function pickFeatures(freeSurfer) {
```

Comment:
We’ll need to update this to pick all features. Returning an array makes `run` choke, unfortunately.